compiler/src/dmd/cparse.d

```diff
@@ -3375,7 +3375,34 @@ final class CParser(AST) : Parser!AST
 
         if (token.value == TOK.identifier)
         {
-            if (token.ident == Id.dllimport)
+            if (token.ident == Id.aligned)
+            {
+                nextToken();
+                if (token.value == TOK.leftParenthesis)
+                {
+                    nextToken();
+                    if (token.value == TOK.int32Literal)
+                    {
+                        const n = token.unsvalue;
+                        if (n < 1 || n & (n - 1) || ushort.max < n)
+                            error("__attribute__((aligned(%lld))) must be an integer positive power of 2", cast(ulong)n);
```

Comment:
There's also the condition that it can't exceed `ushort.max` which the error doesn't mention

Reply:
it does now